go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/fsnotify/fsnotify v1.6.0
 	github.com/gballet/go-libpcsclite v0.0.0-20191108122812-4678299bea08
 	github.com/go-cmd/cmd v1.4.1
+	github.com/golang/mock v1.4.4
```

Comment:
Can we make sure this is using uber's gomock library? I believe we merge this switch into master already so want to make sure we don't depend on both.

Reply:
good catch. weird, I thought I already removed that.